datafiles/static/hackage.css

```diff
@@ -192,24 +202,68 @@ pre {
 .def { font-weight: bold; }
 
 
+/* @end */
+
+
+/* @group responsive */
+
+@media only screen and (min-width: 950px) {
+  #page-header {
+    text-align: left;
+    text-align: left;
+  }
+
+  #page-header ul.links {
+    list-style: none;
+    text-align: left;
+    float: right;
+    display: inline-table;
+    margin: 2px 2em 0 2em;
+  }
+
+  #content {
+    width: 55vw;
+  }
+}
+
+@media only screen and (max-width: 950px) {
+  #page-header {
+    text-align: center;
+  }
+
+  #page-header ul.links {
+    float: none;
+    display: block;
+    text-align: center;
+    margin: 14px 0 0px 0;
+    font-size: 15px;
+  }
+
+  #content {
+    width: 75vw;
```

Comment:
should we however have also `max-width: 1200px` or something like that? Some (accidentally) full screen pages on ridiculously large displays :)

Reply:
Also, imho we *should* have `min-width` which is at wide as narrow viewport close to the breakpoint.

FWIW, Iphone7+ is 1920-by-1080-pixel resolution at 401 ppi, i.e. 960px virtual pixels in landscape, which is wide viewport (as the breakpoint is at 950px).

Reply:
I have added the max-width you suggested. Thanks!

I am not sure about the min-width though. Having a page that floats around as one scroll is annoying, and it doesn't really help reading it. Do you guys have analytics data about how many people use hackage using a small device?